typescript/api/services/I18nEntriesService.ts

```diff
@@ -371,23 +394,30 @@ export module Services {
       const keys = Object.keys(flat);
 
       // Ensure we have a BrandingModel for downstream calls
-      const brandingModel: BrandingModel = (typeof branding === 'string')
-        ? (BrandingService.getBrand(branding) || ({ id: branding } as BrandingModel))
-        : (branding as BrandingModel);
+      const brandingModel: BrandingModel = (BrandingService.getBrandById?.(brandingId)
```

Comment:
When would the methods `BrandingService.getBrandById` or `BrandingService.getBrand` not exist? The optional chaining here implies that those methods may not be present?